include/dfrn.php

```diff
@@ -1853,15 +1859,17 @@ private static function process_relocation($xpath, $relocation, $importer) {
 	 * @param array $item the new item record
 	 * @param array $importer Record of the importer user mixed with contact of the content
 	 * @param int $entrytype Is it a toplevel entry, a comment or a relayed comment?
+	 * @todo set proper type-hints (array?)
 	 */
 	private static function update_content($current, $item, $importer, $entrytype) {
 		$changed = false;
 
 		if (edited_timestamp_is_newer($current, $item)) {
 
 			// do not accept (ignore) an earlier edit than one we currently have.
-			if(datetime_convert("UTC","UTC",$item["edited"]) < $current["edited"])
+			if (datetime_convert("UTC","UTC",$item["edited"]) < $current["edited"]) {
 				return(false);
```

Comment:
`return(false);` is found 16 times, I only fix this now and mark it in todos.txt file.